dev/Vagrantfile

```diff
@@ -118,7 +118,7 @@ def define_chef_server(config, attributes)
       chef.add_recipe("dev::user-env")
       chef.add_recipe("dev::dvm")
       chef.json = json || {}
-      #chef.nodes_path = "nodes"
+      chef.nodes_path = "nodes"
```

Comment:
I actually tried this locally (lastest vagrant) and chef zero failed because of a 404 on the intiial /nodes/ request. I had to change the provider to chef-solo to get dvm online.  That said if it works for you, it's probably a version related thing.  I'll hunt it down and update the README. :+1: